src/components/vice/AccessRequestDialog.js

```diff
@@ -0,0 +1,136 @@
+/**
+ *
+ * @author sriram
+ *
+ */
+import React from "react";
+import { useTranslation } from "i18n";
+import { useMutation } from "react-query";
+import ids from "./ids";
+import { build, announce, FormMultilineTextField } from "@cyverse-de/ui-lib";
+import { Field, Form, Formik } from "formik";
+
+import requestAccess from "serviceFacades/vice/accessRequest";
+import constants from "../../constants";
+
+import DEDialog from "components/utils/DEDialog";
+import { useConfig } from "contexts/config";
+import { Button, CircularProgress, Typography } from "@material-ui/core";
+import ErrorTypographyWithDialog from "components/utils/error/ErrorTypographyWithDialog";
+
+const MIN_USE_CASE_LENGTH = 60;
```

Comment:
This should probably come from configs, so we (and 3rd party installs) can change it on the fly.

Reply:
I can easily add this to configs but I don't see how this value will be changed frequently or vary in different environments / 3rd party installs 🤔 (unlike `concurrent job limit` which needs to be tweaked for resource constraints in different environment / installs) .  As a matter of fact, we don't put other form field validations rules in config.

Reply:
Specifically this value seems somewhat arbitrary to me, and it's a value only high enough to force users asking for VICE access to give us a valid science use case for that access.

I can easily see this gatekeeping value not being high enough to filter out invalid use cases, and we may be asked to increase it. I can also easily see users complaining about a value that is too high, then we'll be asked to reduce it. I can also imagine a 3rd party install not wanting to require this text at all, and they'll want this to be `0`.

It seems a bit heavy to force a rebuild of the software for an arbitrary value like this, which could be changed on the fly by a config setting instead.

Reply:
I added this to config. IMO managing our growing number of configs is more painful than this occasional / rare builds. Also don't we do builds on a config change ? Asking for my own knowledge.

Reply:
That's a good question, and I'm not sure 😬 
If so, I would think that defeats the purpose of configs 🤔